src/Drawable.js

```diff
@@ -515,11 +515,9 @@ class Drawable {
      * @param {int} r The red value of the color, in the range [0,255].
      * @param {int} g The green value of the color, in the range [0,255].
      * @param {int} b The blue value of the color, in the range [0,255].
-     * @param {int} a The alpha value of the color, in the range [0,255].
      * @returns {int} The ID represented by that color.
      */
-    // eslint-disable-next-line no-unused-vars
-    static color4bToID (r, g, b, a) {
+    static color4bToID (r, g, b) {
```

Comment:
I don't speak Complicated Math or scratch-render, but I wonder if this should be color3btoID..?